contrib/docker/README.md

```diff
@@ -0,0 +1,91 @@
+# zcash Dockerfile
+
+This is an example Dockerfile to run the zcashd daemon in a containerized Debian base OS. The image pulls the apt package built by Electric Coin Company.
+
+The output of building this image should be accessible at https://hub.docker.com/repository/docker/electriccoinco/zcashd
+
+## Defaults
+
+The image will run as a non-root user, `zcashd` with uid `2001`. When mapping volumes from the host into the container, these permissions must be used, or rebuilt the image with your custom values.
```

Comment:
“rebuilt the image” -> “rebuild the image” or “the image rebuilt”

Reply:
Thanks, updated.